### PR TITLE
API documentation for DbDataSource

### PR DESCRIPTION
**Pull request by roji**
Closes dotnet/runtime#73722

### CODE REVIEW

File: xml/System.Data.Common/DbDataSource.xml [hunk 1]
```diff
@@ -21,8 +21,8 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents a data source which can be used to obtain open connections, and against which commands can be executed directly.</summary>
+    <remarks>Unlike <see cref="T:System.Data.Common.DbProviderFactory" />, which is purely a factory for provider-specific objections such as <see cref="T:System.Data.Common.DbConnection" />, <see cref="T:System.Data.Common.DbDataSource" /> hands out open connections to specific databases, encapsulating all the information needed to connect to a database. Depending on how the <see cref="T:System.Data.Common.DbDataSource" /> was configured, the connections it hands out may be pooled.</remarks>
```
Comment (David-Engel) [inline xml/System.Data.Common/DbDataSource.xml:25]:
```suggestion
    <remarks>Unlike <see cref="T:System.Data.Common.DbProviderFactory" />, which is purely a factory for provider-specific objects such as <see cref="T:System.Data.Common.DbConnection" />, <see cref="T:System.Data.Common.DbDataSource" /> hands out open connections to specific databases, encapsulating all the information needed to connect to a database. Depending on how the <see cref="T:System.Data.Common.DbDataSource" /> was configured, the connections it hands out may be pooled.</remarks>
```

Comment (roji) [reply xml/System.Data.Common/DbDataSource.xml:25]: Thanks @David-Engel!

**Review by David-Engel — APPROVED** [review]
LGTM

**Review by ajcvickers — APPROVED** [review]

File: xml/System.Data.Common/DbDataSource.xml [hunk 2]
```diff
@@ -126,9 +144,18 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns a <see cref="T:System.Data.Common.DbBatch" /> that's ready for execution against the <see cref="T:System.Data.Common.DbDataSource" />.</summary>
```
Comment (bgrainger) [inline xml/System.Data.Common/DbDataSource.xml:147]: The descriptions for `<Member MemberName="CreateConnection">` and `<Member MemberName="CreateDbBatch">` are switched in this file.

Fix: https://github.com/dotnet/dotnet-api-docs/pull/8319

**Pull request merged**